File: C++/fibonacci.cpp
// Fibonacci Series in C++

#include <iostream>

using namespace std;
void fib(int n)
{
    int t1=0;
    int t2=1;
    int nextterm;
    for(int i=1;i<=n;i++){
        cout<<t1<<" ";
        nextterm=t1+t2;
        t1=t2;
        t2=nextterm;
    }
    return;
}

int main()
{
    fib(10);
}

// Output

// 0 1 1 2 3 5 8 13 21 34

File: Python/fibonacci.py
# python program to print fibonacci series

    # your code goes here
def fibonacci(n):
    a, b = 0, 1
    count = 0

    if n<=0:
        return
    elif n==1:
        print(a)
        return
    else:
        while count<n:
            print(a, end=' ')
            s = a + b
            a = b
            b = s
            count+=1
        
    # print the fibonacci series
    pass


n = int(10)

fibonacci(n)

# Output:

# 0 1 1 2 3 5 8 13 21 34
